tools/flash-shelly.py

```diff
@@ -644,24 +629,55 @@ def device_scan(hosts, action, dry_run, quiet_run, silent_run, mode, exclude, ve
   parser.add_argument('-y', '--assume-yes', action="store_true", dest='silent_run', default=False, help="Do not ask any confirmation to perform the flash.")
   parser.add_argument('-V', '--version',type=str, action="store", dest="version", default=False, help="Force a particular version.")
   parser.add_argument('-c', '--hap-setup-code', action="store", dest="hap_setup_code", default=False, help="Configure HomeKit setup code, after flashing.")
+  parser.add_argument('-v', '--verbose', action="store", dest="verbose", choices=['0', '1'], default='0', help="Enable verbose logging 0=info / 1=debug.")
```

Comment:
0 is not a good default because there should be a way to decrease verbosity as well as increase.
decreasing makes sense if user want no output - makes sense with `-a -y`, for exampel - just upgrade evrything and don't bother me.
so, i suggest we have a level of verbosity for each of the logging levels - trace, debug, info, warning, error, critical (5, ,4, 3, 2, 1, 0 respectively) and set default to 3 (info).

Reply:
ok done.